content/release-notes/2023-05-19-console.md

```diff
@@ -0,0 +1,20 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- Control Plane: Neon's Autoscaling feature now supports fractional (1/4 and 1/2) minimum compute sizes. Previously, the minimum compute size for Autoscaling was 1.
+![Autoscaling fractional compute sizes](/docs/relnotes/fractional_compute_sizes.png)
+  
+  Autoscaling is a [Neon Pro plan](/docs/introduction/pro-plan) feature. For configuration instructions, see [Compute size and Autoscaling configuration](../manage/endpoints#compute-size-and-autoscaling-configuration).
+- API: Added a `synthetic_storage_size` property to the [Projects](https://api-docs.neon.tech/reference/getproject) API, which provides the Neon data storage size in bytes. The value represents  the logical data size plus the amount of Write-Ahead Log (WAL) data stored by Neon. This property is  experimental.
+- UI: Added a **Billing** page to the Neon Console where Free Tier users can  view [Free Tier](/docs/introduction/technical-preview-free-tier) limits and learn about the benefits of upgrading to the [Neon Pro plan](/docs/introduction/pro-plan). The **Billing** page is accessible from the sidebar in the Neon Console.
+![Free Tier Billing page](/docs/relnotes/free_tier_billing.png)
+- UI: The Neon Console now supports **Dark** mode. A selector at the top right of the console allows you to choose from **System**, **Light**, or **Dark** modes.
```

Comment:
this won't be released on prod until later
for now we're going to test it on staging and define all the missing styles